Dockerfile

```diff
@@ -33,7 +33,7 @@ ADD . /opt/code
 RUN chown -R uid1000: /opt
 
 WORKDIR /opt/code/feinstaub
-USER root
+USER uid1000
```

Comment:
Any advantage of running as non-root _in a container_ env?

Reply:
Originally feinstaub did this separation. I am just returning to the original behavour. I think they did this to separate access to log and static files. I can see most of them live in the uid1000 directory which is owned by uid1000.